experimental/traffic-portal/src/app/core/topologies/topology-details/topology-details.component.html

```diff
@@ -0,0 +1,44 @@
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<mat-card appearance="outlined" class="page-content single-column">
+	<tp-loading *ngIf="loading"></tp-loading>
+	<form ngNativeValidate (ngSubmit)="submit($event)">
+		<mat-card-content class="container">
+			<mat-form-field>
+				<mat-label>Name</mat-label>
+				<input matInput type="text" name="name" required [readonly]="!new" [disabled]="!new" [value]="topology.name">
```

Comment:
Why aren't we letting people change Topology names? They aren't immutable:
```shellsession
$ topost -kp topologies '{"name": "dev", "nodes": [{"cachegroup": "dev"}]}'
{
	"alerts": [
		{
			"text": "topology was created.",
			"level": "success"
		}
	],
	"response": {
		"description": "",
		"name": "dev",
		"nodes": [
			{
				"cachegroup": "dev",
				"parents": null
			}
		],
		"lastUpdated": "2023-07-31 22:12:18+00"
	}
}
$ toput -kp 'topologies?name=dev' '{"name": "dev-1", "nodes": [{"cachegroup": "dev"}]}'
{
	"alerts": [
		{
			"text": "topology was updated.",
			"level": "success"
		}
	],
	"response": {
		"description": "",
		"name": "dev-1",
		"nodes": [
			{
				"cachegroup": "dev",
				"parents": null
			}
		],
		"lastUpdated": "2023-07-31 22:13:11+00"
	}
}
```

Reply:
That was unintentional, made the `name` field mutable in 3cb6065082.